src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/books';
+import categorieReducer from './categories/categories';
+
+const rootReducer = () => combineReducers({
+  bookReducer,
+  categorieReducer,
+});
+
+// eslint-disable-next-line no-unused-vars
+const store = configureStore({ reducer: rootReducer });
```

Comment:
- [ ] [OPTOPNAL] Good job creating the store, also you need to export it so you can use it in a provider later, and kindly remove the comment that ignore eslint

Reply:
@Ismailco it was required to pass linter error of store not being used yet, but sure just once I use it I will remove it, thanks.